playbook/lib/generators/kit/templates/kit_html.erb.tt

```diff
@@ -1,8 +1,12 @@
-<%%= pb_content_tag(:div,
-    aria: object.aria,
-    class: object.classname,
-    data: object.data,
-    id: object.id,
-    **combined_html_options) do %>
+# Go to pb_content_tag definition in kit_base.rb for usage information. Commented out options are default (showing the default shape), and each can be deleted if not customizing that param. 
```

Comment:
because this is an html file i don't think this technically is commented out you probalby have to do <!-- --> for stuff out of erb tags. But i like all the words

Reply:
Yep ran into that locally - both lines have been updated to <%# comment %> locally just not yet pushed. Good catch and thanks for noting it!